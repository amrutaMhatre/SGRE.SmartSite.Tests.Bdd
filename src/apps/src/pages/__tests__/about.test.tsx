import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import toJSON  from 'enzyme-to-json';
import About from "../about";

let wrapper: ShallowWrapper;
beforeEach(() => {
    wrapper = shallow(<About />);
});

describe("unit test for About Page", () => {
    test("snapshot test for About page", () => {
        expect(toJSON(wrapper)).toMatchSnapshot();
    })

    test("test for header", () => {
        const title = wrapper.find("h1").text();
        expect(title).toContain("About page");
    })
});

