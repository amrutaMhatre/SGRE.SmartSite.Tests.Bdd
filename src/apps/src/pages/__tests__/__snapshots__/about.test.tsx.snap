// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unit test for About Page snapshot test for About page 1`] = `
<Layout>
  <h1>
    About page
  </h1>
</Layout>
`;
